test/ducks/apps/components/__snapshots__/uninstallModal.spec.js.snap

```diff
@@ -143,7 +143,7 @@ exports[`UninstallModal component should handle correctly linked app error 1`] =
         className="u-mh-half"
         disabled={true}
         extension="full"
-        icon="trash"
+        icon={[Function]}
```

Comment:
I recommend mocking the Trash for this snap and verify trash is present in the snap